nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -60,11 +64,13 @@ def triage_patient(self):
         status = self.triage_task.execute()
         if status == Task.ERROR:
             raise NotImplementedError()
-
-        if status == TriageTask.TIMEOUT:
+        elif status == UserInputs.WD_TIMEOUT:
+            # WD timeout
+            self.phases.put(self.go_idle)
+        elif status == UserInputs.DISMISS:
```

Comment:
Nit: Use or instead of two elif

Reply:
I was thinking we might want different handling for the two options. For now nothing will be implemented but maybe something like an alert can be posted if the watchdog times out?